conf/application.conf

```diff
@@ -98,7 +98,7 @@ countryCodesCsvFilename = "code-lists/mdg-country-codes.csv"
 countryCodesJsonFilename = "code-lists/location-autocomplete-canonical-list.json"
 eu-country-codes-filename = "code-lists/mdg-country-codes-eu.csv"
 draft.timeToLive = "30d"
-pagination.itemsPerPage = 50
+pagination.itemsPerPage = 10
```

Comment:
Was that a product team decision to change to 10 or is this for testing only?

Reply:
Good catch, this was just for testing. I'll revert it.